fetch_gazebo_demo/scripts/pick_place_demo.py

```diff
@@ -167,7 +167,7 @@ def updateScene(self):
             self.scene.addSolidPrimitive(obj.name,
                                          obj.primitives[0],
                                          obj.primitive_poses[0],
-                                         wait = False)
+                                         use_service = False)
```

Comment:
```suggestion
                                         )
```

Reply:
Sorry for the late response, didnt get this update for some reason on my mail. So you want me to do two things basically change:
use_service = False --> use_service = True
And put the parenthesis in another line?